backend/config/config.yaml

```diff
@@ -1,9 +1,5 @@
-db:
-  host: host.docker.internal
-  name: quay
-  port: 5432
-  user: quay
-  password: quay
+DATABASE_SECRET_KEY: '30060361640793187613697366923211113205676925445650250274752125083971638376224'
```

Comment:
Assuming this is not used elsewhere

Reply:
[This is pulled form Quay.](https://github.com/quay/quay/blob/master/local-dev/stack/config.yaml#L8) It's just a random string.